tests/src/test/java/alluxio/hadoop/contract/FileSystemContractCreateIntegrationTest.java

```diff
@@ -14,8 +14,6 @@
 import alluxio.LocalAlluxioClusterResource;
 
 import org.apache.hadoop.conf.Configuration;
-import org.apache.hadoop.fs.contract.AbstractContractCreateTest;
-import org.apache.hadoop.fs.contract.AbstractFSContract;
```

Comment:
Ooops, this is a wrong refactoring generated by IntellJ. Never mind, closing the PR.